### ISSUE TITLE
Public IP label can not have uppercase characters

### ISSUE THREAD

**Issue by sozercan** [openshift/openshift-azure#553]
Running into this issue if someone tries to create a FQDN that includes uppercase characters

```
Deployment failed. Correlation ID: e9fb74af-b2b3-4e8e-a27b-dee4739415ad. Provisioning of resource(s) for container service jzOSACluster in resource group jzOSACluster failed. Message: The domain name label jzOSACluster is invalid. It must conform to the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$.. Details: []
```

**Comment by jim-minter** [comment]
@sozercan is this a dupe of #523, #532?

**Comment by sozercan** [comment]
I don't believe we discussed this before. In RFC 1123 vs 952, we discussed not starting with a number.
and #523 is router fqdn, not `properties.fqdn`.

Idealy, if plugin can convert fqdn to lowercase, that would be great.

**Comment by jim-minter** [comment]
If azure doesn't accept capital letters, we shouldn't either - I think the fix here is to fail to validate with upper case characters.

**Comment by sozercan** [comment]
That works too

**Comment by jim-minter** [comment]
@sozercan I think this issue can be closed - do you?

**Comment by sozercan** [comment]
yes, I believe fixed in #532. thanks!
